src/hooks/use-play-card-audio.ts

```diff
@@ -63,14 +69,30 @@ export function usePlayCardAudio() {
     setActiveCardSide('front');
     await playCardAudio(frontAudio);
 
+    // Start capturing speech
+    const capturedSpeechPromise = captureSpeech(lessonCard.card.back.language as any);
+
     // wait before flip
     const backDuration = await backAudio.getDuration();
     await wait(getPauseLength(backDuration));
 
+    // Stop capturing speech if not already ended
+    stopCapturingSpeech();
+
+    // Currently if the user pauses while the recording is going, we will only get the results back before the pause.
+    // I think this is reasonable for now, but we might want to come up with a better system in the future.
+    const capturedSpeech = await capturedSpeechPromise;
```

Comment:
what would the ideal behavior be?

Reply:
hmmm, after thinking about it, this might be best. 

I was thinking we could restart the card if they paused during speech recognition, but that creates inconsistent behavior with the pause and play buttons. 

I guess we could restart the listening, then combine the results, but there might be some serious challenges with that.